### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

Fix typo

## How to test

Dev deploys

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by nvms — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -204,7 +204,6 @@ commands:
               --var SMTP_SECURE=${<< parameters.smtp_secure >>} \
               --var SMTP_IGNORE_TLS=${<< parameters.smtp_ignore_tls >>} \
               --var FROM_EMAIL_ADDRESS=${<< parameters.from_email_address >>}
-              --var SUPPRESS_ERROR_LOGGING=${<< parameters.suppress_error_logging >>}
```
Comment (GarrettEHill) [inline .circleci/config.yml:207]: Keep this line and add the backslash to the proceeding line at the end

Comment (thewatermethod) [reply .circleci/config.yml:207]: I saw the syntax error earlier, fixing that yields this: https://app.circleci.com/pipelines/github/HHS/Head-Start-TTADP/12536/workflows/31760245-6ce2-495f-a22c-d0f86b3565ad/jobs/70422

Comment (thewatermethod) [reply .circleci/config.yml:207]:
```
Incorrect Usage: invalid argument for flag `--var' (expected []template.VarKV): Expected var 'SUPPRESS_ERROR_LOGGING=' to specify non-empty value
```

Comment (thewatermethod) [reply .circleci/config.yml:207]: I can add it to cloud.gov first to bypass that error but I'm still not sure why we'd need to add it to this particular command

Comment (thewatermethod) [reply .circleci/config.yml:207]: I mean Circle Ci, not Cloud.gov

Regardess, if you want to take ownership of this and tweak it a different way, I'll hold off merging until you can properly start your work day. Turns out we are waiting on demos to deploy to Prod regardless

Comment (kryswisnaskas) [reply .circleci/config.yml:207]: Even though this is a non-secret variable, it will be easier to turn it on/off if we have it in CircleCI

Comment (thewatermethod) [reply .circleci/config.yml:207]: Does it need to be passed to cf deploy then, is that what you mean?

**Review by GarrettEHill — APPROVED** [review]

**Pull request merged**